src/lib/token_list_gen.ts

```diff
@@ -76,106 +127,203 @@ export const generateTokenList = async (
   const name = l1TokenList.name
   const mainLogoUri = l1TokenList.logoURI
 
-  const { l1 , l2 } = await getNetworkConfig();
+  const { l1 , l2 } = await promiseErrorMultiplier(getNetworkConfig(), (error) => getNetworkConfig(), 3, false);
 
-  let tokens =
+  let tokens: GraphTokenResult[] =
     options && options.getAllTokensInNetwork
-      ? await getAllTokens(l2.network.chainID)
-      : await getTokens(
+      ?  await promiseErrorMultiplier(getAllTokens(l2.network.chainID), (error) => getAllTokens(l2.network.chainID), 3, false)
+      : await promiseErrorMultiplier(
+        getTokens(
           l1TokenList.tokens.map((token) => ({
             addr: token.address.toLowerCase(),
             logo: token.logoURI
           })),
           l2.network.chainID
-        );
-
+        ),
+        (error) => getTokens(
+          l1TokenList.tokens.map((token) => ({
+            addr: token.address.toLowerCase(),
+            logo: token.logoURI
+          })),
+          l2.network.chainID
+        ), 
+        3, 
```

Comment:
seems like we use this value everywhere, lets add it as a default value to simplify calling the `promiseErrorMultiplier`